apps/api/pages/api/event-types/[id]/_get.ts

```diff
@@ -51,7 +51,6 @@ export async function getHandler(req: NextApiRequest) {
     include: {
       customInputs: true,
       team: { select: { slug: true } },
-      users: true,
```

Comment:
@keithwillcode we should select fields instead of removing. This will omit participants from the api response